src/pages/home/HeroSection.jsx

```diff
@@ -0,0 +1,33 @@
+import { Link } from "react-router-dom";
+import styles from "./Home.module.css";
+import "../../styles/utils.css";
+
+export default function HeroSection() {
+  return (
+    <>
```

Comment:
No need to use react fragments , when you are returning a single div.

Reply:
Yes that's correct but I am returning a single parent component hence why I'm using react fragments